src/DataSource/Http.hs

```diff
@@ -34,6 +35,7 @@ defaultConfig =
         , cfgZendesk            = "https://iohk.zendesk.com"
         , cfgToken              = ""
         , cfgEmail              = "daedalus-bug-reports@iohk.io"
+        , cfgGroupId            = GroupId 0
```

Comment:
Remove this. We don't require this much flexibility for now. We just need the _admin_ group for now.